modules/_UXTeam.scss

```diff
@@ -0,0 +1,104 @@
+
+#UXTeam{
+    text-align: center;
+
+.tittle{
+        padding: 2rem 0;
+        color:#000;
+        font-size: var(--size-6);
+        font-weight: 700;
+        line-height: 1.15;
+        .detail{
+            color:var(--orange-color);
+        }
+}
+p{
```

Comment:
What are we selecting here? if you want "globally" to change all paragraphs inside the section, at least move it to the top and not between specific elements